cmd/gitops/beta/run/cmd.go

```diff
@@ -539,6 +540,19 @@ func runCommandWithoutSession(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(cfg)
+	if err != nil {
+		return err
+	}
+
+	serverVersion, err := discoveryClient.ServerVersion()
+	if err != nil {
+		return err
+	}
+
+	// We need the server version to pass to the validation package
+	kubernetesVersion := strings.TrimPrefix(strings.TrimRight(serverVersion.GitVersion, "+"), "v")
```

Comment:
Is this supposed to turn e.g. "1.25.1+gke1234" into "1.25"? Because I don't think it does.

Reply:
Thanks for spotting this!

test cases added.